modules/ROOT/pages/provision-adapter-configure-service-mesh-CLI.adoc

```diff
@@ -35,17 +35,18 @@ You can delete an adapter if it is no longer required. However, you must delete
 
 When you provision an adapter, or bind the adapter to an application, a message is displayed after you run an adapter command. The message indicates whether the adapter was created, deleted, or modified successfully. 
 
-Additionally, you can verify the adapter status, which provides further details about the adapter. 
-
-To view the adapter status, type the following command:
-
+Additionally, you can verify the adapter status, which provides further details about the adapter. To view the adapter status, type the following command:
++
 `./service-mesh adapter list`
 
 The status values include:
 
-* Ready--The adapter is now ready to be used. 
-* Failed--The adapter was not created successfully. An error occurred. 
-* Provisioning--The adapter is in the process of being provisioned.
+* Ready: The adapter is now ready to be used. 
+* Failed: The adapter was not created successfully. An error occurred. 
+* Provisioning: The adapter is in the process of being provisioned.
+* DeprovisionBlockedByExisingCredentials: The adapter has existing bindings, which are in use. 
```

Comment:
I think the list of values should be in backticks to avoid possible translation: `Ready`, `Failed`, `Provisioning`, `DeprovisionBlockedByExisingCredentials`

Reply:
Really? Kim didn't mention that. So now I'm wondering. Let me check with her.